styles.css

```diff
@@ -0,0 +1,92 @@
+.toolbar-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 48px;
+  padding: 0 12px;
+}
+
+#logo {
+  width: 76px;
+  height: 20px;
+  margin-top: 10px;
+}
+
+#nav-logo {
+  width: 16px;
+  height: 20px;
+  margin-right: 20px;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 100%;
+  height: 604px;
+  top: 92px;
+}
+
+#name {
+  height: auto;
+  width: 80%;
+  margin-left: 24px;
+  margin-top: 22px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+#introduction {
+  height: 146px;
+  width: 80%;
+  margin-left: 24px;
+  margin-top: 18px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+#connect {
+  height: 24px;
+  width: 31.2%;
+  margin-left: 24px;
+  margin-top: 12px;
+}
+
+#social {
+  display: flex;
+  justify-content: flex-start;
+  margin-left: 24px;
+  width: fit-content;
+  list-style: none;
+  padding: 0;
+}
+
+#social li {
+  margin-right: 10px;
+}
+
+#social li a {
+  display: block;
+}
+
+#social li a img {
+  width: 24px;
+  height: 24px;
+}
+
+body {
+  background: url(./images/background.png);
+  background-repeat: no-repeat;
+  background-size: 100% 718px;
```

Comment:
- [ ] Although I appreciate 🎉🎉👌your effort in adding the background image as suggested by the previous reviewer, there are still some improvements needed for mobile responsiveness within the range of 375px to 768px. 
- [ ] Currently, your design does not align with the suggested [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=XeKkHNg24zP2w1kF-0) design. 
- [ ] To ensure a consistent and visually appealing experience across different devices, it is important to make the necessary adjustments to your design.🤔🤔
- [ ] Your Design:
![Screenshot 2023-06-03 042123](https://github.com/mhassa204/Portfolio-Project/assets/118761303/292c30b6-4fab-4f74-a0f8-a9dc31df6a1e)

- [ ] Figma Design:
![figma](https://github.com/ngalamac/mobile-portfolio/assets/118761303/30ac202c-97ab-4ab0-859b-f9a788b0da16)

Reply:
Please explain this part.